Preparing related docs for MWL GA (#12008)
DataDog/documentation: content/en/metrics/summary.md

{{< img src="metrics/summary/bulkpercentiles.jpg" alt="Bulk Percentiles"  style="width:75%;">}}

* **Configure Tags**: Configure tags on multiple custom metrics matching a namespace using Metrics without Limits™

{{< img src="metrics/summary/bulkconfig.gif" alt="Bulk Metric Tag Configuration"  style="width:75%;">}} 

## Metric details sidepanel

Click on any metric name to display its details sidepanel for more information regarding the metric’s metadata and tags: 

{{< img src="metrics/summary/mwl_sidepanel.jpg" alt="Metric panel"  style="width:75%;">}}

### Metric name

The name of your metric in the [Metrics Explorer][2], [dashboards][3], etc.

### Ingested Custom Metrics

A metric name may emit multiple ingested custom metrics depending on its associated tag value combinations. Ingested custom metrics represent all the data originally submitted via code.

Learn more in the [custom metrics][4] documentation.

> ### Indexed Custom Metrics
> Unlike ingested custom metrics, indexed custom metrics represent those that remain queryable across the Datadog platform. This number may be impacted by adding/removing percentile aggregations or use of Metrics without Limits. Learn more in the [Metrics without Limits][10] documentation

Reviewer: ```suggestion
### Indexed Custom Metrics

Unlike ingested custom metrics, indexed custom metrics represent those that remain queryable across the Datadog platform. This number may be impacted by adding or removing percentile aggregations or by use of Metrics without Limits™. Learn more in the [Metrics without Limits™][10] documentation.
```

- Open a graph of this metric filtered by your tag `key:value` pair in the Metrics Explorer.
- Copy any tag `key:value` for filtering across the application.

{{< img src="metrics/summary/updated_tags_table.gif" alt="Tags Table"  style="width:75%;">}}

[Learn more about tagging][5].

## Metrics without Limits\*

> Metrics without Limits provides you control over the size of your custom metrics without requiring any agent/code-level changes (more details here).  Note: Metrics without Limits is only available for custom metrics.

Reviewer: ```suggestion
Metrics without Limits\* provides you control over the size of your custom metrics without requiring any agent or code-level changes. 

**Note:** Metrics without Limits\* is only available for custom metrics.
```

> You can configure tags using our bulk metric tag configuration button or the **Manage Tags** button in a metric's details sidepanel. 

Reviewer: ```suggestion
You can configure tags using the bulk metric tag configuration button or the **Manage Tags** button in a metric's details side panel. 
```

{{< img src="metrics/distributions/managetags.png" alt="Configuring tags on a distribution"  style="width:80%;">}}

> 1. Click on your custom distribution metric name in the Metrics Summary table to open the metrics details sidepanel.

Reviewer: ```suggestion
1. Click on your custom distribution metric name in the **Metrics Summary** table to open the metrics details side panel.
```

2. Click the **Manage Tags** button to open the tag configuration modal.
3. Click the **Custom...** tab to customize the tags you'd like to keep available for query. Tag configurations are _allowlists_ of the tags you'd like to keep. 

> 4. Preview the effects of your proposed allowlist with the cardinality estimator before hitting Save.

Reviewer: ```suggestion
4. Preview the effects of your proposed allowlist with the cardinality estimator before selecting **Save**.
```

> **Note**: The exclusion of tags is not supported in the allowlist-based customization of tags. Adding tags starting with `!` is not accepted. And the cardinality estimator requires the metric to be 48 hours and older.

Reviewer: ```suggestion
**Note**: The exclusion of tags is not supported in the allowlist-based customization of tags. Adding tags starting with `!` is not accepted. Also, the cardinality estimator requires the metric to be older than 48 hours.
```

> ### Queryable Tags 
> Once your metric has been configured with Metrics without Limits\*, you can view which tags remain Queryable -- ultimately those that contribute to _Indexed Custom Metrics_ volume. And you can toggle back to all originally submitted and ingested tags that contribute to your _Ingested Custom Metrics_ volume. 

Reviewer: ```suggestion
### Queryable tags 

Once your metric has been configured with Metrics without Limits\*, you can view which tags remain Queryable -- ultimately those that contribute to _Indexed Custom Metrics_ volume. And you can toggle back to all originally submitted and ingested tags that contribute to your _Ingested Custom Metrics_ volume. 
```

> ### Fine-tune your metric with aggregations in Advanced Mode
> For custom metrics of the count/gauge/rate metric type, you can further fine-tune your metric's configurations by optionally including additional aggregations with the advanced mode of Metrics without Limits\*. By default, Datadog stores the most frequently queried aggregation combination depending on the metric's type to preserve the mathematical accuracy of your configured metric's query as listed below: 

Reviewer: ```suggestion
### Fine-tune your metric with aggregations in Advanced Mode

For custom metrics of the count, gauge, or rate metric type, you can further refine your metric's configurations by optionally including additional aggregations with the advanced mode of Metrics without Limits\*. By default, Datadog stores the most frequently queried aggregation combination depending on the metric's type to preserve the mathematical accuracy of your configured metric's query as listed below: 
```

> - Configured counts/rates will be queryable with time/space aggregations of `SUM`

Reviewer: ```suggestion
- Configured counts/rates are queryable with time/space aggregations of `SUM`
```

> - Configured gauges will be queryable in time/space aggregations of `AVG`

Reviewer: ```suggestion
- Configured gauges are queryable in time/space aggregations of `AVG`
```

> {{< img src="metrics/summary/customize_aggr_docs.jpg" alt="Fine-tune aggregations on counts/rates/gauges"  style="width:80%;">}}

Reviewer: ```suggestion
{{< img src="metrics/summary/customize_aggr_docs.jpg" alt="Refine aggregations on counts, rates, and gauges"  style="width:80%;">}}
```

> More aggregations are available should they be valuable to you. You can add or remove aggregations at any time with no required Agent or code-level changes. **Note**: Configuring your count/rate/gauge metric and removing an aggregation may impact existing dashboards and monitors.

Reviewer: ```suggestion
More aggregations are available should they be valuable to you. You can add or remove aggregations at any time with no required Agent or code-level changes.

**Note**: Configuring your count, rate, or gauge metric and removing an aggregation may impact existing dashboards and monitors.
```
